pandas/core/series.py

```diff
@@ -1096,9 +1096,26 @@ def __setitem__(self, key, value) -> None:
             if com.is_bool_indexer(key):
                 key = check_bool_indexer(self.index, key)
                 key = np.asarray(key, dtype=bool)
+
```

Comment:
This is from #44275?

Reply:
yes (mentioned in OP)

Reply:
Yeah, thx. Was a bit confused why I could not see that master was updated here